modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -5,7 +5,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-ingress-configuring-application-domain_{context}"]
-= Configuring application domain for the Ingress Controller Operator
+= Configuring cluster domain for applications using appsdomain option.
```

Comment:
You could change "Configuring cluster domain" to "Configuring a cluster domain".

Reply:
Acked